ifcbdb/dashboard/accession.py

```diff
@@ -262,8 +262,8 @@ def import_metadata(metadata_dataframe, progress_callback=do_nothing):
     df = metadata_dataframe.copy()
 
     BIN_ID_COLUMNS = ['id','pid','lid','bin','bin_id','sample','sample_id','filename']
-    LAT_COLUMNS = ['latitude','lat','y']
-    LON_COLUMNS = ['longitude','lon','lng','lg','x']
+    LAT_COLUMNS = ['latitude','lat','y','gptLatitude']
```

Comment:
this is misspelled, should be `gpsLatitude`

Reply:
All set